conseil-api/src/main/scala/tech/cryptonomic/conseil/api/ConseilApi.scala

```diff
@@ -194,7 +200,13 @@ class ConseilApi(config: CombinedConfiguration)(implicit system: ActorSystem)
         case Failure(exception) => logger.error("Pre-caching attributes failed", exception)
         case Success(_) => logger.info("Pre-caching attributes successful!")
       }
-    } else logger.warn("Pre-catching could not be done, because all of the block-chains are disabled!")
+    } else {
+      throw NoNetworkEnabledError("""
+        |Pre-catching could not be done, because all of the block-chains are disabled!
+        | It means that application is NOT configured properly.
+        | Please double check configuration file and start API again.
+        |""".stripMargin)
```

Comment:
Sorry to be pedantic on the wording, but I might suggest some tweaks, and correct a few typos

>Pre-caching can't be done, because there is no enabled block-chain defined.
This probably means the application is NOT properly configured.
The API needs a platform section to be defined, which lists at least one enabled chain and network.
Please double-check the configuration file and start the API service again.

Reply:
I appreciate the intent, nonetheless, the spirit was correct

Reply:
Don't be sorry :) I've updated that.